app/uk/gov/hmrc/thirdpartyapplication/config/Scheduler.scala

```diff
@@ -36,13 +36,14 @@ class Scheduler @Inject()(upliftVerificationExpiryJobConfig: UpliftVerificationE
                           metricsJob: MetricsJob,
                           bcryptPerformanceMeasureJob: BCryptPerformanceMeasureJob,
                           hashExistingClientSecretsJob: HashExistingClientSecretsJob,
+                          updateClientSecretNamesJob: UpdateClientSecretNamesJob,
                           apiStorageConfig: ApiStorageConfig,
                           app: Application) extends RunningOfScheduledJobs {
 
   override val scheduledJobs: Seq[ExclusiveScheduledJob] = {
     val upliftJob = if (upliftVerificationExpiryJobConfig.enabled) Seq(upliftVerificationExpiryJob) else Seq.empty
     val mJob = if (metricsJobConfig.enabled) Seq(metricsJob) else Seq.empty
-    val bcryptJobs = Seq(bcryptPerformanceMeasureJob, hashExistingClientSecretsJob)
+    val bcryptJobs = Seq(bcryptPerformanceMeasureJob, hashExistingClientSecretsJob, updateClientSecretNamesJob)
```

Comment:
Do we need all these jobs?

Reply:
I'm not sure if the hashing job has gone all the way through to Prod yet, so I left it in to be safe. It will come out as part of the ticket to stop storing the cleartext secrets

Reply:
Going to leave the performance job in at least until we're on Future platform. Will let us see how that performs...